src/main/java/net/jodah/expiringmap/ExpiringMap.java

```diff
@@ -1382,4 +1382,15 @@ public V setValue(V value) {
       }
     };
   }
+
+  public synchronized static void shutdown() {

```

Comment:
Can we get some javadoc here indicating what this method does?

Reply:
Sure, will add it as soon as I'm back in the office.

Reply:
(sorry, I was on other projects and time has flown by)